server-phoenix/lib/helios/widget.ex

```diff
@@ -34,4 +38,11 @@ defmodule Helios.Widget do
     tod = DateTime.to_time(time)
     from q in query, where: (q.start <= ^tod or is_nil(q.start)) and (q.stop >= ^tod or is_nil(q.stop))
   end
+
+  def next_or_default(query, current_id) do
+    queryable = from q in query, where: (q.id == ^current_id), select: q.position
+    current_position = Helios.Repo.one(queryable)
+    w = from q in query, where: (q.position > ^current_position)
+    w |> first
```

Comment:
Does this method return the default if `current_id` is missing or `current_id` is the last widget?